### DISCUSSION TITLE
(Feature Request) Individual search history removal

### DISCUSSION THREAD

**Discussion by eunwoo1104** [koreanbots/koreanbots#361] [제안]
![image](https://user-images.githubusercontent.com/61371424/117547121-e4fff880-b068-11eb-9b3f-e94486ee89dd.png)
최근 검색어에서 특정 기록만 지울 수 있도록 하는 것도 지원하면 좋을 것 같습니다.

**Comment by wonderlandpark** [comment]
해당 기능은 v2.1 이상에서 서포트 해보겠습니다.
